services/src/main/java/org/keycloak/workflow/admin/resource/WorkflowResource.java

```diff
@@ -34,6 +38,11 @@ public WorkflowResource(WorkflowProvider provider, Workflow workflow) {
     }
 
     @DELETE
+    @Tag(name = KeycloakOpenAPI.Admin.Tags.REALMS_ADMIN)
```

Comment:
For workflows we probably want to add a new WORKFLOWS tag, and use it in all endpoint methods instead of REALM_ADMIN